host-interaction/process/inject/use-pe-injection.yml

```diff
@@ -0,0 +1,29 @@
+rule:
+  meta:
+    name: use pe injection
```

Comment:
propose: `inject PE`

Reply:
"this function may...."
"inject *a* PE"

Reply:
Fixed in aa94a33